cartography/intel/azure/sql.py

```diff
@@ -87,7 +86,7 @@ def get_server_details(credentials, subscription_id, server_list):
 def get_dns_aliases(credentials, subscription_id, server):
     try:
         client = get_client(credentials, subscription_id)
-        dns_aliases = list(client.server_dns_aliases.list_by_server(server['resourceGroup'], server['name']))
+        dns_aliases = client.server_dns_aliases.list_by_server(server['resourceGroup'], server['name']).as_dict()['value']
```

Comment:
azure sdk performs the 'value' to dns alias conversion as part of the list_by_server function call. We should use the same code as you have used in "get_server_list" function. use lambda to generate dict of dns alias objects.